src/components/Home/README.md

```diff
@@ -0,0 +1,13 @@
+# `<Home />`
+
+OVERVIEW_HERE
+
+## Example
+
+![Home](../../../.loki/reference/chrome_Home_default.png)
```

Comment:
We'll need to update the image paths in our README.md template

Reply:
https://github.com/cremalab/crema-app-web/pull/17/commits/70932653596b1a79373c89f32188162d64536f4c